buddyboss/buddyboss-platform
# Photo more options hover, photo popup shows previous image for a while

### All Submissions:


Fixes #2082 

### How to test the changes in this Pull Request:

1. Check photos (...) ellipsis dropdown hover
2. Photos Popup should not show the last opened image and then new select image.
3. View Group Photos in Popup as Organizer >Now order is: “Report” just before “delete” button

### Proof Screenshots or Video



### Other information:
